src/HPRT/Analytics/views.py

```diff
@@ -136,6 +183,26 @@ def get_context_data(self, **kwargs):
             ]
         ]
 
+        g1_DSMV_avg = self.find_avg_scores(list(map(self.get_patient_data, map(lambda x: list(DSMV.objects.filter(patient=x.id)), list(g1_patients)))))
+        g2_DSMV_avg = self.find_avg_scores(list(map(self.get_patient_data, map(lambda x: list(DSMV.objects.filter(patient=x.id)), list(g2_patients)))))
+        g1_DSMV_x = [i * 6 for i in (list(range(0, len(g1_DSMV_avg))))]
```

Comment:
As before, that value of 6 looks like it should be some sort of constant/parameter